sympy/solvers/tests/test_inequalities.py

```diff
@@ -486,3 +495,156 @@ def test__pt():
     assert _pt(x, oo) == _pt(oo, x) == x + 1
     assert _pt(x, -oo) == _pt(-oo, x) == x - 1
     raises(ValueError, lambda: _pt(Dummy('i', infinite=True), S.One))
+
+
+def test_linprog_maximize():
+    np = import_module("numpy")
+    scipy = import_module("scipy")
+    linprog = scipy.optimize.linprog
```

Comment:
Is it okay for most of the tests to use SciPy? That would mean that a lot of tests would get skipped if SciPy is not installed.